case-lib/lib.sh

```diff
@@ -47,11 +47,13 @@ func_lib_setup_kernel_last_line()
 SOF_LOG_COLLECT=0
 func_lib_start_log_collect()
 {
-    local is_etrace=${1:-0} ldcFile
-    ldcFile=/etc/sof/sof-$(sof-dump-status.py -p).ldc || {
-        >&2 dlogw "sof-dump-status.py -p to query platform failed"
-        return
-    }
+    local is_etrace=${1:-0} ldcFile="$SOFLDC"
+    if [[ -z "$SOFLDC" ]]; then
+        ldcFile=/etc/sof/sof-$(sof-dump-status.py -p).ldc || {
+            >&2 dlogw "sof-dump-status.py -p to query platform failed"
```

Comment:
This error message doesn't say what's missing (the ldcfile) and it doesn't suggest any solution.

Reply:
@marc-hb  If $SOFLDC is empty,  we will try to use /etc/sof/sof-\*.ldc, (default sof-\*.ldc folder in CI), I think we should add error message/solution after `ldcFile=/etc/sof/sof-$(sof-dump-status.py -p).ldc || { >&2 dlogw "sof-dump-status.py -p to query platform failed" `}, before I make any change, I want to get input from you.

I will remove SOFLDC in config.sh.

`$(sof-dump-status.py -p)` will give the platform name, eg, cml, apl, etc.